etc/OAuth2_Config.pm

```diff
@@ -127,7 +156,7 @@ Set(%OAuthIDPs,
             Lang => 'locale',
             Organization => 'hd',
         },
-        'LoginPageButton' => '/static/images/btn_google_signin_dark_normal_web.png',
+        'LoginPageButton' => '/NoAuth/images/btn_google_signin_dark_normal_web.png',
```

Comment:
Any reason to change this? I believe images are stored in /static

Reply:
It wasn't working for me otherwise.  Maybe a 4.0 thing.